inc/class_captcha.php

```diff
@@ -125,6 +135,16 @@ function __construct($build = false, $template = "")
 			{
 				$this->build_recaptcha();
 			}
+		}else if(in_array($this->type, array(6, 7)) && $mybb->settings['captchapublickey'] && $mybb->settings['captchaprivatekey'])
```

Comment:
Nitpick: code style:

```suggestion
		}
		elseif(in_array($this->type, array(6, 7)) && $mybb->settings['captchapublickey'] && $mybb->settings['captchaprivatekey'])
```

Reply:
Thanks, done :)